upgrading/automated_upgrades.adoc

```diff
@@ -636,255 +405,170 @@ openshift_node_upgrade_post_hook=/usr/share/custom/post_node.yml
-re-deployed. If so, manually change the DeploymentConfig and DaemonSet to
-`imagePullPolicy: Always` to make sure it is re-pulled.
-====
-
-[[upgrading-cluster-metrics]]
-== Upgrading Cluster Metrics
-
-To upgrade an existing cluster metrics deployment, you must use the provided
-*_/usr/share/ansible/openshift-ansible/playbooks/openshift-metrics/config.yml_*
-Ansible playbook. This is the playbook to use if you were deploying metrics for
-the first time on an existing cluster, but is also used to upgrade existing
-metrics deployments.
-
-. If you have not already done so, see
-xref:../install_config/cluster_metrics.adoc#metrics-ansible-variables[Specifying
-Metrics Ansible Variables] in the
-xref:../install_config/cluster_metrics.adoc#install-config-cluster-metrics[Enabling
-Cluster Metrics] topic and update your Ansible inventory file to at least set
-the following required variables within the `[OSEv3:vars]` section:
-+
-----
-[OSEv3:vars]
-
-openshift_metrics_install_metrics=true <1>
-openshift_metrics_hawkular_hostname=<fqdn> <2>
-openshift_metrics_cassandra_storage_type=(emptydir|pv|dynamic) <3>
-----
-<1> Enables the ability to upgrade the metrics deployment.
-<2> Used for the Hawkular Metrics route. Specify a fully qualified
-domain name.
-<3> Choose the same type as the previous deployment.
-
-. Add any other `openshift_metrics_*` variables that you want to specify to
-override the defaults, as described in
-xref:../install_config/cluster_metrics.adoc#metrics-ansible-variables[Specifying
-Metrics Ansible Variables].
+. After all master and node upgrades have completed, reboot all hosts.
 
-. When you have finished updating your inventory file, follow the instructions
-in
-xref:../install_config/cluster_metrics.adoc#deploying-the-metrics-components[Deploying
-the Metrics Deployment] to run the *_openshift-metrics/config.yml_* playbook and
-complete the metrics deployment upgrade.
+. If you use aggregated logging, xref:upgrading-efk-logging-stack[upgrade the EFK logging stack].
 
-[[special-considerations-for-mixed-environments]]
-== Special Considerations for Mixed Environments
+. If you use cluster metrics, xref:upgrading-cluster-metrics[upgrade cluster metrics].
 
-Mixed environment upgrades (for example, those with Red Hat Enterprise Linux and
-Red Hat Enterprise Linux Atomic Host) require setting both
-`openshift_pkg_version` and `openshift_image_tag`. In mixed environments,  if
-you only specify `openshift_pkg_version`, then that number is used for the
-packages for Red Hat Enterprise Linux and the image for Red Hat Enterprise
-Linux Atomic Host.
+. xref:verifying-the-upgrade[Verify the upgrade].
 
 [[special-considerations-for-glusterfs]]
-== Special Considerations When Using Containerized GlusterFS
-When upgrading  {product-title}, you must upgrade the set of nodes where
-GlusterFS pods are running.
+== Upgrading {product-title} when using containerized GlusterFS
```

Comment:
Do these instructions still work? I'd like to update this section to explain which playbooks to run, but I am not sure how to upgrade the masters, then the infrastructure, then the individual nodes. Would you run a control plane upgrade and then manually upgrade each node in the cluster?